### ISSUE TITLE
New rule: ARIA required owned elements

### ISSUE THREAD

**Issue by ajanec01** [w3c/wcag-act#468]
The ACT-Rules community group has completed work on [ARIA required owned elements](https://act-rules.github.io/rules/bc4a75). We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: ARIA required owned elements
**Rule id**: bc4a75
**Link to the rule**: [ARIA required owned elements](https://act-rules.github.io/rules/bc4a75)
**License**: W3C Software And Document Notice And License
**Implementors**:
- QualWeb
- SortSite

**Comment by maryjom** [comment]
Created [survey](https://www.w3.org/2002/09/wbs/93339/ACTRuleARIArequiredOwns/) due 6 August.

**Comment by tbostic32** [comment]
# Feedback
Rule: https://act-rules.github.io/rules/bc4a75
Survey results: https://www.w3.org/2002/09/wbs/93339/ACTRuleARIArequiredOwns/results

## Trevor

- [ ] Passed example 4 invalid html. `<li>` element not under under any of`<li>, <ul>, <menu>`. Elements with `role="menuitemradio"` and `role="menuitemcheckbox"` don't have required attribute `aria-checked`.

- [ ] Notes 1 and 3 could be moved into the background section. 

## Kathy

- [ ] Element has implicit semantic role that is identical to its explicit semantic role. Why is this exempt in the applicability?

**Comment by tbostic32** [comment]
My first comment on the HTML is being addressed in this pull request: https://github.com/act-rules/act-rules.github.io/pull/1414

From our discussion the invalid HTML is intended and should be covered by the accessibility support section.

Kathy's comment is currently under discussion in issue https://github.com/act-rules/act-rules.github.io/issues/1413
